ios/RNDeviceInfo/RNDeviceInfo.m

```diff
@@ -511,6 +512,12 @@ - (NSDictionary *)powerState
     resolve(@(isCharging));
 }
 
+RCT_EXPORT_METHOD(hasLocationServicesEnabled:(RCTPromiseResolveBlock)resolve rejecter:(RCTPromiseRejectBlock)reject)
+{
+    BOOL locationServicesEnabled = [CLLocationManager locationServicesEnabled];
```

Comment:
Thinking along the lines of return object shape, if it wasn't a simple boolean we could easily expose the rest of the availability items here as well: https://developer.apple.com/documentation/corelocation/determining_the_availability_of_location_services - a lot of those are more related to abilities as opposed to whether something is on or not, so the wrapper that allowed for API consumers to also just have a simple yes/no would have to be a little more picky for iOS, but in general just returning a map of those should be easy? (I'm more used to android where it's just a WritableMap with some pushStrings and  you're set